spec/features/unsubscribing_spec.rb

```diff
@@ -4,37 +4,31 @@
   end
 
   scenario "unsubscribing from an email uuid, then no longer receiving emails" do
-    Timecop.freeze "2019-06-26 10:00" do
-      login_with_internal_app
+    login_with_internal_app
+    subscriber_list_id = create_subscriber_list
+    subscribe_to_subscriber_list(subscriber_list_id, frequency: "daily")
 
-      subscriber_list_id = create_subscriber_list
-      subscribe_to_subscriber_list(subscriber_list_id, frequency: "daily")
-
-      Timecop.freeze "2019-06-25 09:30:00" do
-        create_content_change(public_updated_at: "2019-06-25 10:00:00")
-      end
+    travel_to(Time.zone.yesterday.midday) { create_content_change }
 
+    travel_to(Time.zone.today.midday) do
       DailyDigestInitiatorWorker.new.perform
       Sidekiq::Worker.drain_all
+    end
 
-      email_data = expect_an_email_was_sent
-
-      id = extract_unsubscribe_id(email_data)
-      unsubscribe_from_subscriber_list(id, expected_status: 204)
-
-      clear_any_requests_that_have_been_recorded!
+    email_data = expect_an_email_was_sent
+    id = extract_unsubscribe_id(email_data)
+    unsubscribe_from_subscriber_list(id, expected_status: 204)
+    clear_any_requests_that_have_been_recorded!
 
-      Timecop.freeze "2019-06-25 09:30:01" do
-        create_content_change(public_updated_at: "2019-06-25 10:00:00")
-      end
+    travel_to(Time.zone.today.midday) { create_content_change }
```

Comment:
```suggestion
    travel_to(Time.zone.today.midnight) { create_content_change }
```

I think the purpose of the test is to check more recent content changes are excluded; this would be a little clearer if we could change the data, but keep the frozen runtime the same.